src/http-journeys.md

```diff
@@ -765,3 +765,152 @@ remainingCount | Number | Total amount of statements left to process
   "isPublic": true,
 }
 ```
+
+# Journey Recalculation HTTP Interface
+This interface provides API access to the function we use to recalculate the journey. Requests to this interface should look something like the request below.
+
+```http
+POST http://www.example.org/api/recalculatejourney?journeyId=dfb7218c-0fc9-4dfc-9524-d497097de027
+Content-Type: application/json
+Authorization: <YOUR_BASIC_AUTH_TOKEN>
+```
+
+The interface will respond with a response similar to the one below after start of recalculation a matching journey.
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json; charset=utf-8
+
+{
+    "journey": {
+        "recalculateStatus": {
+            "pushing": true,
+            "inProgress": true,
+            "totalCount": 0,
+            "remainingCount": 0,
+            "recalculatedAt": "2020-12-14T10:29:22.439Z",
+            "cancelledAt": "2020-12-10T09:53:45.929Z",
+            "cancelledBy": "5fd0d4de5fa8774c995b2258",
+            "recalculatedBy": "5fd0d4de5fa8774c995b2258",
```

Comment:
These seem to be the old attributes, please confirm if they are still needed @PrinceWaune ?